gnome-initial-setup/gis-driver.c

```diff
@@ -87,33 +87,55 @@ typedef struct _GisDriverPrivate GisDriverPrivate;
 
 G_DEFINE_TYPE_WITH_PRIVATE(GisDriver, gis_driver, GTK_TYPE_APPLICATION)
 
+/* Should be kept in sync with eos-installer */
 static gboolean
-running_live_session (void)
+check_for_live_boot (gchar **uuid)
```

Comment:
This parameter is not optional because the function is listed verbatim from `eos-installer` where the UUID is always required. It is also always required here, so I don't see the advantage of making it optional!